lib/src/frame.dart

```diff
@@ -21,7 +21,7 @@ final _v8Frame =
     RegExp(r'^\s*at (?:(\S.*?)(?: \[as [^\]]+\])? \((.*)\)|(.*))$');
 
 // http://pub.dartlang.org/stuff.dart.js:560:28
-final _v8UrlLocation = RegExp(r'^(.*):(\d+):(\d+)|native$');
+final _v8UrlLocation = RegExp(r'^(.*?):(\d+)(?::(\d+))?|native$');
```

Comment:
I apologize, when I made the earlier comment I didn't realize that the column/line was extracted here and not in `_v8EvalLocation`, now I see why you had the separate regexp initially. I was hoping that we could ignore the column only in the context of an eval, but I see now that it is much trickier with the code as it is written below.

The current solution looks good. Just a couple questions:
* can we omit the extra `?` on the first group?
* if it is the same below, we could also write this with an additional `|` to list the three cases separately:
  `^(.*):(\d+):(\d+)|native|(.*):(\d+)$`. If you find that more readable, feel free to switch.

Reply:
This won't work without extra `?` because of greedy match.